.github/workflows/tests.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - name: Install Django ${{ matrix.django-version }}
         run: poetry run pip install "django==${{ matrix.django-version }}"
       - name: Test with pytest
-        run: poetry run pytest --showlocals -vvv --cov-report=xml
+        run: poetry run pytest -n auto --showlocals -vvv --cov-report=xml
```

Comment:
Not 100% sure if this is effective in the runner instance, as it won't be helpful running this on a single core machine.